build-docker.sh

```diff
@@ -8,7 +8,7 @@ set -x
 
 GPU=""
 BASE_IMAGE="ubuntu:focal"
-WHEEL_URL="https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp37-cp37m-manylinux2014_x86_64.whl"
+WHEEL_URL="https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.8.0-cp37-cp37m-manylinux2014_x86_64.whl"
```

Comment:
this change is kind of wrong. we do not save 2.8.0 on `latest/` prefix. maybe revert.

Reply:
what prefix do we save it on? I notice the previous commit had this change too

Reply:
- it will be something like `https://s3-us-west-2.amazonaws.com/ray-wheels/releases/2.8.0/<commit>/ray-2.8.0-cp38-cp38-manylinux2014_x86_64.whl`
- we do not have py37 wheels any more for 2.8.1
- latest seems to be saving wheels built from master head
- this `WHEEL_URL` will almost always be overwritten when being used.